tests/stress/cases/modules/deploy/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "layer0_deploy" "base" {
```

Comment:
As stated in the previous pull request, I think we should attempt to use only a single Terraform configuration for all stress tests, with "entity-per-environment" input parameters.

Reply:
Use of `random_pet` to generate names and proper leveraging of `element()` has resulted in a single-file terraform configuration! (Excluding `variables.tf` and `outputs.tf`, that is.)

✅